src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -412,8 +412,8 @@ const renderAuctionHighlight = artist => {
 }
 
 const renderRepresentationStatus = artist => {
-  const { highlights } = artist
-  const { partnersConnection } = highlights
+  const { artistHightlights } = artist
```

Comment:
These `artistHighlight` changes get our VSCode error panel back to a healthy state /cc @sweir27

Reply:
Thank you! Sorry for dropping the ball on that one.